lib/gauge_fix_fft.cu

```diff
@@ -19,17 +19,12 @@ namespace quda {
 
 #ifdef GPU_GAUGE_ALG
```

Comment:
What type of correctness checking was done for gauge fixing? Just the standard test executable?

Reply:
Yes, which checks most of the code paths in question here but certainly not all.  The gauge fixing code is one of those parts of the code that is overly due for a rewrite (using accessors) and deployment into ctest to improve coverage.  Obviously beyond nature of this pull which is just reformatting `<< <` into `<<<` and deleting the old texture reference code path.